packages/@sanity/core/src/actions/dataset/listDatasetCopyJobs.js

```diff
@@ -0,0 +1,82 @@
+module.exports = async function listDatasetCopyJobs(args, context) {
+  const {apiClient, output, chalk} = context
+  const flags = args.extOptions
+  const client = apiClient()
+  const projectId = client.config().projectId
+  const query = {}
+  let response
+
+  if (flags.offset && flags.offset >= 0) {
+    query.offset = flags.offset
+  }
+  if (flags.limit && flags.limit > 0) {
+    query.limit = flags.limit
+  }
+
+  try {
+    response = await client.request({
+      method: 'GET',
+      uri: `/projects/${projectId}/datasets/copy`,
+      query,
+    })
+  } catch (error) {
+    if (error.statusCode) {
+      output.print(`${chalk.red(`Dataset copy list failed:\n${error.response.body.message}`)}\n`)
+    } else {
+      output.print(`${chalk.red(`Dataset copy list failed:\n${error.message}`)}\n`)
+    }
+  }
+
+  if (response && response.length > 0) {
+    output.print('Dataset copy jobs for this project:')
+    const print = []
+
+    response.forEach((job) => {
+      const {id, state, createdAt, updatedAt, sourceDataset, targetDataset, withHistory} = job
+
+      const createdAtInSeconds = new Date(createdAt).getTime() / 1000
+      const nowInSeconds = new Date().getTime() / 1000
+      const timePassedInSeconds = nowInSeconds - createdAtInSeconds
+      const timeStarted = convertTimestampToDaysHoursMinsSecs(timePassedInSeconds)
+
+      const updatedAtInSeconds = new Date(updatedAt).getTime() / 1000
+      const timeTakenInSeconds = updatedAtInSeconds - createdAtInSeconds
+      const timeTaken = convertTimestampToDaysHoursMinsSecs(timeTakenInSeconds)
+
+      print.push({
+        jobId: id,
+        state: state,
+        history: withHistory,
+        'time started': `${timeStarted} ago`,
+        'time taken': timeTaken,
+        'source dataset': sourceDataset,
+        'target dataset': targetDataset,
+      })
+    })
+
+    output.printTable(print)
```

Comment:
We should provide an order to the rows in the table.
```suggestion
    output.printTable(print, ['jobId', 'source dataset', 'target dataset', 'state', 'history', 'time started', 'time taken'])
```